nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -267,4 +268,21 @@ def get_compress_pipeline(
 class OVAWQAlgoAlgoBackend(OVWeightCompressionAlgoBackend):
     @staticmethod
     def get_awq_patterns():
-        return get_awq_patterns(om.OVMatMulMetatype, om.OVMultiplyMetatype)
+        return get_awq_patterns(om.OVMatMulMetatype, om.OVMultiplyMetatype, ATOMIC_ACTIVATIONS_OPERATIONS)
+
+    @staticmethod
+    def insert_scale_after_node(node, scale, node_name):
```

Comment:
It looks like, this is standard transformation. Could you use 
```
OVCommandCreator.multiply_insertion_command(
            source_node, nodes, source_output_port_id, scale_value, scale_node_name
        )
```
https://github.com/openvinotoolkit/nncf/blob/65c9515989e3bcef46145baa25de3d4106289eeb/nncf/quantization/algorithms/smooth_quant/openvino_backend.py#L114C15-L114C59

Reply:
Done